lib/nunjucks/govuk-prototype-kit/layouts/govuk-branded.html

```diff
@@ -1,4 +1,6 @@
-{%- set assetPath = '/extension-assets/govuk-frontend/govuk/assets' -%}
+{% if GOVUKPrototypeKit.legacyGovukFrontendFixesNeeded %}
+  {%- set assetPath = '/extension-assets/govuk-frontend/govuk/assets' -%}
+{% endif %}
```

Comment:
I think this was correct before, `assetPath` always needs to be set.

Reply:
You're right, updated.